core-api/src/main/java/com/optimizely/ab/config/parser/JsonSimpleConfigParser.java

```diff
@@ -156,13 +156,11 @@ public ProjectConfig parseProjectConfig(@Nonnull String json) throws ConfigParse
             Condition conditions = null;
             if (experimentObject.containsKey("audienceConditions")) {
                 Object jsonCondition = experimentObject.get("audienceConditions");
-                if (jsonCondition instanceof JSONArray) {
-                    try {
-                        conditions = ConditionUtils.<AudienceIdCondition>parseConditions(AudienceIdCondition.class, (JSONArray)jsonCondition);
-                    } catch (Exception e) {
-                        // unable to parse conditions.
-                        Logger.getAnonymousLogger().log(Level.ALL, "problem parsing audience conditions", e);
-                    }
+                try {
+                    conditions = ConditionUtils.<AudienceIdCondition>parseConditions(AudienceIdCondition.class, jsonCondition);
```

Comment:
@nchilada this is the only other one that will expect any type.  audience.conditions as we mentioned in the ticket and offline will remain legacy.